docs/Users_Guide/load_data.rst

```diff
@@ -1,333 +1,725 @@
-**********
-Background
-**********
+***************
+Using METdbLoad
+***************
 
-The METdbLoad module provides support for inserting MET output data into the database.
+The METdbLoad module provides support for inserting MET output data (as .stat files) into a relational database
+(mysql, mariadb, or aurora).
 
-Before using the METdbLoad module, the database must exist and have the proper permissions
-(i.e. grant privileges to insert, delete, update, and index).  A schema file, *mv_mysql.sql* is available in the
-METdataio/METdbLoad/sql/ directory for importing into the database.
 
-The METdbLoad script *met_db_load.py* performs the loading of data based on settings in an XML specification file.
+Create Database
+---------------
+
+Before using the METdbLoad module, the database **must** exist and have the proper permissions
+(i.e. grant privileges to insert, delete, update, and index).
+
+
+Data must be loaded into a database which has the prefix \'\mv_\'\,
+(e.g. mv_met_data). This database must be structured with the METviewer
+mv_mysql.sql schema:
+
+.. code-block:: ini
+
+  cd [install]/metviewer
+  mysql -u[db_username] -p[db_password] -e'create database [db_name];'
+  mysql -u[db_username] -p[db_password] [db_name] < sql/mv_mysql.sql
+
+
+- Replace *db_usenamer* with the username
+
+- Replace *db_passwd* with the password
+
+- Replace *db_name* with the appropriate database name that begins with **mv** (e.g. mv_xyz )
+
+
+.. note::
+ Use the mysql command line client (https://dev.mysql.com/doc/refman/en/mysql.html) to run the above instructions.
+
+ The *mv_mysql.sql* file is located in the METdataio/METdbLoad/sql/ directory.
 
-In the METdataio/METdbLoad/sql/scripts directory, there are two configuration files:
+.. dropdown:: Create MET linetype tables
 
-  * db_load_specification.xml
+ mysql -udbuser -pdbpasswd dbtable_name < /path-to-METdataio/METdbLoad/sql/mv_mysql.sql
 
-  * data_loading_config.yaml
 
-The *db_load_specification.xml* is a template XML specification file, and *data_loading_config.yaml*
-is a template YAML configuration file.  The *data_loading_config.yaml* file contains
-information about the database (username, password, database name, etc.). This information is used by the
-*generate_xml_spec.py* script to generate the XML specification file which is then used to load data into the database.
 
-Generate the XML Specification File
+    - Replace *path-to-METdataio* with the path where the METdataio source code is saved
```

Comment:
```suggestion
    - Replace *dbuser* with the username
    - Replace *dbpasswd* with the password
    - Replace *dbtable_name* with the appropriate database name that begins with **mv** (e.g. mv_xyz )
    - Replace *path-to-METdataio* with the path where the METdataio source code is saved
```
It looks like we lost the other entries that we had previously, I think.  Please correct me if I am wrong.

Reply:
@bikegeek this is in the Create MET linetype tables section.

Reply:
@bikegeek You should be able to click "Commit suggestion" here if you agree.